### PR TITLE
[PATH-2] Create ER diagram in draw.io7

### PR DESCRIPTION
**Pull request by swornim00**
Routes added

Fixes: https://github.com/dotnepal/blood-donors/issues/2

### CODE REVIEW

File: resources/views/layouts/app.blade.php [hunk 1]
```diff
@@ -1,6 +1,87 @@
...
+                </div>
+
+                <div class="collapse navbar-collapse" id="app-navbar-collapse">
+                    <!-- Left Side Of Navbar -->
+                    <ul class="nav navbar-nav">
+                        &nbsp;
+                    </ul>
+
+                    <!-- Right Side Of Navbar -->
+                    <ul class="nav navbar-nav navbar-right">
```
Comment (samundra) [inline resources/views/layouts/app.blade.php]: create navigation template partial.

File: resources/views/layouts/app.blade.php [hunk 2]
```diff
@@ -1,6 +1,87 @@
...
+                    </ul>
+                </div>
+            </div>
+        </nav>
+
+        @yield('content')
+    </div>
+
+    <!-- Scripts -->
+    <script src="/js/app.js"></script>
```
Comment (samundra) [inline resources/views/layouts/app.blade.php]: use elixir

File: routes/web.php [hunk 3]
```diff
@@ -12,3 +12,7 @@
 */
 
 Route::get('/', 'SiteController@index');
+
+Auth::routes();
```
Comment (samundra) [inline routes/web.php]: use auth middleware don't define route yourself for it.

File: resources/views/auth/passwords/email.blade.php [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+                                        <strong>{{ $errors->first('email') }}</strong>
+                                    </span>
+                                @endif
+                            </div>
+                        </div>
+
+                        <div class="form-group">
+                            <div class="col-md-6 col-md-offset-4">
+                                <button type="submit" class="btn btn-primary">
+                                    Send Password Reset Link
```
Comment (samundra) [inline resources/views/auth/passwords/email.blade.php]: use translations

File: resources/views/auth/passwords/email.blade.php [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+                        <div class="alert alert-success">
+                            {{ session('status') }}
+                        </div>
+                    @endif
+
+                    <form class="form-horizontal" role="form" method="POST" action="{{ url('/password/email') }}">
+                        {{ csrf_field() }}
+
+                        <div class="form-group{{ $errors->has('email') ? ' has-error' : '' }}">
+                            <label for="email" class="col-md-4 control-label">E-Mail Address</label>
```
Comment (samundra) [inline resources/views/auth/passwords/email.blade.php]: use translations

File: resources/views/auth/passwords/email.blade.php [hunk 6]
```diff
@@ -0,0 +1,47 @@
+@extends('layouts.app')
+
+<!-- Main Content -->
+@section('content')
+<div class="container">
+    <div class="row">
+        <div class="col-md-8 col-md-offset-2">
+            <div class="panel panel-default">
+                <div class="panel-heading">Reset Password</div>
```
Comment (samundra) [inline resources/views/auth/passwords/email.blade.php]: use translations

File: app/Http/Controllers/HomeController.php [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+class HomeController extends Controller
+{
+    /**
+     * Create a new controller instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        $this->middleware('auth');
```
Comment (samundra) [inline app/Http/Controllers/HomeController.php]: use auth middleware in the route, don't use here.

**Review by samundra — CHANGES_REQUESTED** [review]
Please address the comments

File: resources/views/layouts/app.blade.php (hunk 1 above)
Comment (swornim00) [reply resources/views/layouts/app.blade.php]: Do you mean adding creating one .blade.php file for navigation?

**Comment by samundra** [comment]
Please commit changes into separate PRs. That way it's easier to rollback if need be and is more manageable.

**Pull request closed without merging**